### PR TITLE
Update README and move start.sh script (and rename)

### PR DESCRIPTION
**Pull request by leo-scalingo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,21 +121,31 @@ This is the equivalent of:
 ip addr del MY_IP dev MY_INTERFACE
 ```
 
-## Dev environment
+## Development environment
```
Comment (EtienneM) [inline README.md:124]: praise: I like this kind of attention to details :star_struck:

File: README.md [hunk 2]
```diff
@@ -121,21 +121,31 @@ This is the equivalent of:
...
-```shell
-modprobe dummy
-ip link add eth10 type dummy
-ip link set eth10 up
-ip link add eth11 type dummy
-ip link set eth11 up
-ip link add eth12 type dummy
-ip link set eth12 up
 ```
+for idx in 10 11 12 ; do
```
Comment (EtienneM) [inline README.md:132]: suggestion: maybe explicitly mention that this code block is `bash` to make sure it's correctly colored?

**Review by EtienneM — APPROVED** [review]
Just a suggestion otherwise LGTM :)

File: README.md [hunk 3]
```diff
@@ -121,21 +121,31 @@ This is the equivalent of:
...
 
-```shell
-modprobe dummy
-ip link add eth10 type dummy
-ip link set eth10 up
-ip link add eth11 type dummy
-ip link set eth11 up
-ip link add eth12 type dummy
-ip link set eth12 up
 ```
```
Comment (leo-scalingo) [inline README.md:131]:
```suggestion
```bash
```

**Pull request merged**